Mixpanel/Mixpanel+HostWatchOS.m

```diff
@@ -14,7 +14,7 @@ @implementation Mixpanel (HostWatchOS)
 - (void)session:(WCSession *)session didReceiveMessage:(NSDictionary<NSString *, id> *)message {
     NSString *messageType = [Mixpanel messageTypeForWatchSessionMessage:message];
     if (messageType) {
-        if ([messageType isEqualToString:@"track"]) {
+        if ([messageType isEqualToString:@"track:properties:"]) {
```

Comment:
@Quuxplusone Totally meant `NSStringFromSelector` updated. I actually think it would be worthwhile to change many of those selectors as well. 

Although I agree with many points of the article you linked, I think we can all agree the "stringly" typed variants we are using now are not buying us anything.